src/index.js

```diff
@@ -1,34 +1,23 @@
 import './style.css';
+import { addTodo, removeTodo, completeTodo } from './modules/addTodo';
```

Comment:
- [ ] Within the requirements of this milestone, A function for editing task descriptions should  be implemented. Please consider adding this function. Currently tasks' description cannot be updated.

Reply:
What you exactly mean with edit? the localstorage or the array with the description, and how it supposed to be the workflow to edit an entry? I don't remember have seen that in the video.